src/plots.jl

```diff
@@ -1,3 +1,39 @@
+# gives domain length along a specified direction
+function domain_length(sim, direction::Symbol)
```

Comment:
This exact function is in SDFResults: https://github.com/SebastianM-C/SDFResults.jl/blob/master/src/files/utils.jl#L13